db_mongo/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mongo:4.4.9
```

Comment:
We cannot provide out own image to the cluster registry, so we should use publicly available database image and initialize it separetly

Reply:
For production we can use the fill script on the db that's already running in the cluster (our db images are not meant to be used in prod), while for development we can use these small niceties and just have it fill automatically on compose up. Sure, the discrepancy between production and dev is not ideal if we were trying to maintain this project for a long time but since it's just for a student project, it will just make it easier to develop with (and our goal shifted to not overengineering due to limited time 😄).

Is there a reason that makes this a bad idea for our use case or problematic with project requirements?

Reply:
The idea would be to try to match as closely as possible final production environment to avoid any sort of integration hell. During functional tests we need to seed database anyway and they should run exactly as they would on cluster.